src/components/ChapterMap.svelte

```diff
@@ -27,12 +28,14 @@
 
   <legend>
     <div>
-      <span style="background: var(--lightBlue)" /> aktiver Standort
+      <span style="background: var(--lightBlue)" />
+      {$microcopy?.map?.text?.active}
     </div>
     <div>
-      <span style="background: var(--darkGreen)" /> in Gründung
+      <span style="background: var(--darkGreen)" />
+      {$microcopy?.map?.text?.funding}
```

Comment:
https://wikipedia.org/wiki/Funding

You mean `founding` but maybe `in_setup` would be the better term anyway.